includes/Media.php

```diff
@@ -233,8 +236,67 @@ public static function wp_prepare_attachment_for_js( $response, $attachment ) {
 	 *
 	 * @return string[]
 	 */
-	public static function upload_mimes( array $mime_types ) {
-		$mime_types['svg'] = 'image/svg+xml';
+	public static function mime_types( array $mime_types ) {
+		$mime_types['svg']  = 'image/svg+xml';
+		$mime_types['svgz'] = 'image/svg+xml';
 		return $mime_types;
 	}
+
+	/**
+	 * Filter wp_check_filetype_and_ext to allow SVGs.
+	 *
+	 * @param array  $checked File data array containing 'ext', 'type', and 'proper_filename' keys.
+	 * @param string $file                      Full path to the file.
+	 * @param string $filename                  The name of the file (may differ from $file due to
+	 *                                          $file being in a tmp directory).
+	 * @param array  $mimes                     Key is the file extension with value as the mime type.
+	 * @return Array           [description]
```

Comment:
Missing description

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/832/commits/b3cfe7b88cf7782a1033f6a0b71362a0be2afbfd